Update autoscaling_group.html.markdown (#32429)
hashicorp/terraform-provider-aws: website/docs/r/autoscaling_group.html.markdown

    - `instance_warmup` - (Optional) Number of seconds until a newly launched instance is configured and ready to use. Default behavior is to use the Auto Scaling Group's health check grace period.
    - `min_healthy_percentage` - (Optional) Amount of capacity in the Auto Scaling group that must remain healthy during an instance refresh to allow the operation to continue, as a percentage of the desired capacity of the Auto Scaling group. Defaults to `90`.
    - `skip_matching` - (Optional) Replace instances that already have your desired configuration. Defaults to `false`.

>     - `auto_rollback` - (Optional) Automatically rollback if instance refresh fails. Defaults to `false`. Must be using launch_template to enable this option. This is not supported for launch_configuration.

Reviewer: Based on the [AWS documentation](https://docs.aws.amazon.com/autoscaling/ec2/userguide/asg-instance-refresh.html#instance-refresh-how-it-works), it appears that this may also be used when using `mixed_instances_policy`. As such, suggesting slightly different working (and formatted the arguments with backticks).

```suggestion
    - `auto_rollback` - (Optional) Automatically rollback if instance refresh fails. Defaults to `false`. This option may only be set to `true` when specifying a `launch_template` or `mixed_instances_policy`.
```
